emcc.py

```diff
@@ -734,76 +734,77 @@ def get_cflags(options, user_args):
   if cflags:
     return cflags
 
-  # Flags we pass to the compiler when building C/C++ code
-  # We add these to the user's flags (newargs), but not when building .s or .S assembly files
-  cflags = get_clang_flags()
+  with ToolchainProfiler.profile_block('get_cflags'):
```

Comment:
Rather than re-indenting all this code can you create a wrapper function instead so this patch is small?
 
```
def get_cflags(options, user_args):
   with ToolchainProfiler.profile_block('get_cflags'):
      _get_cflags(options, user_args):

def _get_cflags(options, user_args):
   ...
```

Reply:
In future we could do something clever like:

```
@profile("get_cflags")
def get_cflags(options, user_args):
   ...
```